Client.Core/Client.Core.csproj

```
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
```

1. Please change order of versions and add `LangVersion`:
```suggestion
        <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
        <LangVersion>8</LangVersion>
```
2. Add `Microsoft.Bcl.AsyncInterfaces` ass conditional reference:
```xml
<ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' ">
        <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.1" />
</ItemGroup>
```